client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
Why `PascalCase`? What's wrong with just having it as a variable? For cases like this I don't consider them any different because I could just as easily see them as being defined inside of the function body as any other given variable. To change it here would feel like doing this inside of React `render()` bodies…

```js
render() {
	const POSTS = this.props.posts.filter( isPublic );
}
```

Do we really need to add another naming convention? I think it's bad enough that we have our PHP styles permeating Calypso because things aren't ever the way I expect them: it's JavaScript, so why isn't it `camelCase`?

Put another way, do we treat these variables in a significantly different enough way that it's justifiable to name them differently? How do we treat them different? What is their classification in contrast to normal variables or variables defined with `const`?